docs/_docs/user-guide/eldritch.md

```diff
@@ -548,3 +548,33 @@ The <b>crypto.hash_file</b> method will produce the hash of the given file's con
 - SHA1
 - SHA256
 - SHA512
+
+### crypto.encode_b64
+`crypto.encode_b64(content: str, encode_type: str) -> str`
+
+The <b>crypto.encode_b64</b> method encodes the given text using the given base64 encoding method. Valid methods include:
+
+- STANDARD
+- STANDARD_NO_PAD
+- URL_SAFE
+- URL_SAFE_NO_PAD
+
+### crypto.decode_b64
+`crypto.decode_b64(content: str, decode_type: str) -> str`
+
+The <b>crypto.decode_b64</b> method encodes the given text using the given base64 decoding method. Valid methods include:
+
+- STANDARD
+- STANDARD_NO_PAD
+- URL_SAFE
+- URL_SAFE_NO_PAD
+
+### crypto.from_json
+`crypto.from_json(content: str) -> Value`
+
+The <b>crypto.from_json</b> method converts JSON text to an object of correct type.
```

Comment:
Could you include an example with JSON string and what the output would look like if you printed the resulting object